### PR TITLE
#225 one order per month

### PR DESCRIPTION
**Pull request by akoebbe**
This PR only allows one Partner Order and one Distribution Order per month.

This also fixes some icon changes after the FA upgrade

### CODE REVIEW

File: assets/js/views/order/distribution/BulkDistributionOrderEdit.vue [hunk 1]
```diff
@@ -255,6 +301,22 @@
                 axios
                     .delete('/api/orders/distribution/' + this.$route.params.id)
                     .then(self.$router.push('/orders/distribution'));
+            },
+            checkPartnerCanOrder: (val) => {
+                console.log('in handler (val)', val);
+                console.log('in handler', this);
```
Comment (ericpoe) [inline assets/js/views/order/distribution/BulkDistributionOrderEdit.vue:307]: [nb] Are these leftovers from debugging?

File: docker-compose.unison.yml [hunk 2]
```diff
@@ -2,35 +2,19 @@ version: '3.1'
 
 services:
     app:
-        build:
```
Comment (ericpoe) [inline docker-compose.unison.yml:5]: This section is needed in order for xdebug to work. See README section "Debugging with Xdebug". The `env_file` is needed for xdebug, too.

Comment (ericpoe) [reply docker-compose.unison.yml:5]: See #241 where `docker-compose.unison.yml` can finally get this resolved.

File: assets/js/views/order/distribution/BulkDistributionOrderEdit.vue (hunk 1 above)
Comment (akoebbe) [reply assets/js/views/order/distribution/BulkDistributionOrderEdit.vue]: Indeed. Fixed.

**Review by ericpoe — APPROVED** [review]

**Pull request merged**